draft-ietf-ppm-dap.md

```diff
@@ -2066,7 +2045,7 @@ message contains the following parameters:
 
     * For time_interval tasks, the request specifies the batch interval.
 
-    * For fixed_size tasks, the request specifies the batch ID.
+    * For leader_selected tasks, the request specifies the batch ID.
```

Comment:
names with _ are usually backquoted. Consider adding `` to  this and time_interval on above row